Description: profiles: Add attach_disconnected flag to dnsmasq profile
 .
 https://launchpad.net/bugs/1569316
 .
 When Ubuntu made the jump from network-manager 1.0.4 to 1.1.93, the
 dnsmasq process spawned from network-manager started hitting a
 disconnected path denial:
 .
   audit: type=1400 audit(1460463960.943:31702): apparmor="ALLOWED"
     operation="connect" info="Failed name lookup - disconnected path"
     error=-13 profile="/usr/sbin/dnsmasq"
     name="run/dbus/system_bus_socket" pid=3448 comm="dnsmasq"
     requested_mask="wr" denied_mask="wr" fsuid=65534 ouid=0
 .
 Signed-off-by: Tyler Hicks <tyhicks@canonical.com>
 Acked-by: Christian Boltz <apparmor@cboltz.de>
Origin: commit, revision id: tyhicks@canonical.com-20160412213643-yjj14272h3f3dptk
Author: Tyler Hicks <tyhicks@canonical.com>
Bug: https://launchpad.net/bugs/1569316
Last-Update: 2016-04-12
X-Bzr-Revision-Id: tyhicks@canonical.com-20160412213643-yjj14272h3f3dptk

=== modified file 'profiles/apparmor.d/usr.sbin.dnsmasq'
--- old/profiles/apparmor.d/usr.sbin.dnsmasq	2016-04-08 20:23:47 +0000
+++ new/profiles/apparmor.d/usr.sbin.dnsmasq	2016-04-12 21:36:43 +0000
@@ -12,7 +12,7 @@
 @{TFTP_DIR}=/var/tftp /srv/tftpboot
 
 #include <tunables/global>
-/usr/sbin/dnsmasq {
+/usr/sbin/dnsmasq flags=(attach_disconnected) {
   #include <abstractions/base>
   #include <abstractions/dbus>
   #include <abstractions/nameservice>

